pkg/driver/sqlite/sqlite.go

```diff
@@ -131,7 +131,7 @@ func (drv *Driver) schemaMigrationsDump(db *sql.DB) ([]byte, error) {
 // DumpSchema returns the current database schema
 func (drv *Driver) DumpSchema(db *sql.DB) ([]byte, error) {
 	path := ConnectionString(drv.databaseURL)
-	schema, err := dbutil.RunCommand("sqlite3", path, ".schema")
+	schema, err := dbutil.RunCommand("sqlite3", path, ".schema --nosys")
```

Comment:
Pretty sure these should be separate arguments right?
```suggestion
	schema, err := dbutil.RunCommand("sqlite3", path, ".schema", "--nosys")
```

Are you sure this change did anything for you? Since you mentioned the test never failed, in which situations did you see those tables get created?

Reply:
I can recreate this using the steps in the issue just not in the unit test.

If I add the `.schema --nosys` as one string the `sqlite_sequence` does not get dumped to the schema. If I add it as a sepeate string I receive an error.

#### No change applied

```sh
j.doherty@test my_dbmate % DATABASE_URL=sqlite:db/db.sqlite ./dist/dbmate dump
Writing: ./db/schema.sql
j.doherty@test my_dbmate % cat ./db/schema.sql 
CREATE TABLE IF NOT EXISTS "schema_migrations" (version varchar(255) primary key);
CREATE TABLE t (
      id INTEGER PRIMARY KEY AUTOINCREMENT
);
CREATE TABLE sqlite_sequence(name,seq);
-- Dbmate schema migrations
INSERT INTO "schema_migrations" (version) VALUES
  ('20230220094634');
```

#### With my patch applied

```sh
j.doherty@test my_dbmate % DATABASE_URL=sqlite:db/db.sqlite ./dist/dbmate dump
Writing: ./db/schema.sql
j.doherty@test my_dbmate % cat ./db/schema.sql                                
CREATE TABLE IF NOT EXISTS "schema_migrations" (version varchar(255) primary key);
CREATE TABLE t (
      id INTEGER PRIMARY KEY AUTOINCREMENT
);
-- Dbmate schema migrations
INSERT INTO "schema_migrations" (version) VALUES
  ('20230220094634');
```

#### With the flag in a seperate arg

```sh
j.doherty@test my_dbmate % DATABASE_URL=sqlite:db/db.sqlite ./dist/dbmate dump
Error: sqlite3: Error: unknown option: -nosys
Use -help for a list of options.
```

Reply:
I see. That table only shows up in the dump if you have an `AUTOINCREMENT` field. I added one to the test and it correctly fails now.

Reply:
Good spot. Don't know why I didn't think to check the test migration.